rxlifecycle/src/main/java/com/trello/rxlifecycle/RxLifecycle.java

```diff
@@ -33,12 +34,12 @@ private RxLifecycle() {
      * <p>
      * When the lifecycle event occurs, the source will cease to emit any notifications.
      * <p>
-     * Use with {@link Observable#compose(Observable.Transformer)}:
+     * Use with {@link Observable#compose(ObservableTransformer)}:
      * {@code source.compose(RxLifecycle.bindUntilEvent(lifecycle, ActivityEvent.STOP)).subscribe()}
      *
      * @param lifecycle the lifecycle sequence
      * @param event the event which should conclude notifications from the source
-     * @return a reusable {@link Observable.Transformer} that unsubscribes the source at the specified event
+     * @return a reusable {@link ObservableTransformer} that unsubscribes the source at the specified event
```

Comment:
it returns `LifecycleTransformer` that implements `ObservableTransformer` plus the one for Single + Completable